src/playbooks/deploy/metadata.obsah.yaml

```diff
@@ -9,6 +9,19 @@ variables:
     help: Initial password for the admin user.
   foreman_puma_workers:
     help: Number of workers for Puma.
+  external_authentication:
+    help: External authentication method to use
+    choices:
+      - ipa
+      - ipa_with_api
+  ipa_manage_sssd:
+    help: Whether to manage SSSD service and configuration for IPA authentication
+    action: store_true
+  pam_service:
+    help: Name of the PAM service to use for IPA authentication
+  gssapi_local_name:
+    help: Whether to use the local name for GSSAPI authentication
+    action: store_true
```

Comment:
Do we want to explicitly expose those or leave EXTRA_VARS as the only option for overriding those?